### DISCUSSION TITLE
help guide

### DISCUSSION THREAD

**Discussion by ArmoredArtist** [GoMudEngine/GoMud#122] [Ideas]
I am unsure how to interact with the guide. 

Suggestion: add guide to the help file as "Help guide"

New players will want to know what the guide can do and how to dismiss him if hes annoying.  which is can be as seen in the photo :)

![image](https://github.com/user-attachments/assets/74baed14-c4aa-4e23-ab2a-e0a96dfc4acb)

**Comment by ArmoredArtist** [comment]
Armoredartist's Guide says to you, "You can find help on many subjects by typing help."
help
.: Help for help

The help command looks up guidance on a command.

Usage: 

  help say
  Find help on the command say.

Here are some skills and commands to look up to get started:

(nothing is here about guide and help guide does not work)

**Comment by Volte6** [comment]
Closing this, created an issue for it here: https://github.com/Volte6/GoMud/issues/123

**Comment by cgthunderstrike** [comment]
hellllp

**Reply by Volte6** [reply]
:D Whats' up
